services/app-web/src/pages/QuestionResponse/QuestionResponse.tsx

```diff
@@ -52,6 +56,7 @@ export const QuestionResponse = () => {
     return (
         <div className={styles.background}>
             <GridContainer className={styles.container}>
+                <QuestionResponseSubmitBanner submitType={submitType} />
```

Comment:
We have been using the pattern of not rendering components if it's toggled off. I like in `StateDashboard.tsx`

```
  {justSubmittedSubmissionName && (
      <SubmissionSuccessMessage
          submissionName={justSubmittedSubmissionName}
      />
  )}
```
Any particular reason we are not doing that here? Just curious.

Reply:
I will change my code. I like consistency, thanks for pointing it out. Either works, of course.

In terms of why - my thinking was `submitType` is treated as an enum in the banner component and for that reason its not the same as a generic string like submitted name, which is displayed directly in the banner regardless of value as long as it exists.  I wanted to the banner component to handle the business logic around that enum in same place. I just put the undefined handling next that handling what happens if invalid value like `submitType=foobar`  come in the through URL.